src/applications/vre/25-8832/config/form.js

```diff
@@ -0,0 +1,116 @@
+import fullSchema from 'vets-json-schema/dist/28-8832-schema.json';
+import environment from 'platform/utilities/environment';
+import { VA_FORM_IDS } from 'platform/forms/constants';
+import { externalServices } from 'platform/monitoring/DowntimeNotification';
+
+import IntroductionPage from '../containers/IntroductionPage';
+import ConfirmationPage from '../containers/ConfirmationPage';
+import { statusSelection } from './chapters/status-selection';
+import { veteranInformation } from './chapters/veteran-information';
+import GetFormHelp from '../../components/GetFormHelp';
+import PreSubmitInfo from '../components/PreSubmitInfo';
+
+import {
+  claimantInformation,
+  claimantAddress,
+} from './chapters/claimant-information';
+import { isDependent, transform } from './helpers';
+import manifest from '../manifest.json';
+
+import { WIZARD_STATUS } from '../constants';
+
+const formConfig = {
+  rootUrl: manifest.rootUrl,
+  urlPrefix: '/',
+  submitUrl: `${environment.API_URL}/v0/education_career_counseling_claims`,
+  trackingPrefix: 'careers-employment-28-8832--',
```

Comment:
Are you planning on updating this tracking prefix?

Reply:
Not currently unless you feel can include this without impacting other teams. I'm not sure what it's used for.